packages/default/scss/upload/_theme.scss

```
@@ -2,129 +2,89 @@
-                background-color: $upload-progress-bg;
+            .k-progressbar .k-selected {
+                background-color: $kendo-upload-progress-bg;
             }
         }
 
 
         // Upload status: success
         .k-file-success {
-            .k-progress {
-                background-color: $upload-success-bg;
+            .k-file-validation-message {
+                color: $kendo-upload-success-text;
+            }
+
+            .k-progressbar .k-selected {
+                background-color: $kendo-upload-success-bg;
             }
         }
 
         // Upload satus: error
-        .k-file-error,
-        .k-file-invalid {
+        .k-file-error {
             .k-file-validation-message {
-                color: $upload-error-text;
+                color: $kendo-upload-error-text;
             }
-            .k-progress {
-                background-color: $upload-error-bg;
+            .k-progressbar .k-selected {
+                background-color: $kendo-upload-error-bg;
             }
         }
 
-        .k-file-extension-wrapper,
-        .k-multiple-files-extension-wrapper,
-        .k-file-group-wrapper,
-        .k-multiple-files-group-wrapper,
-        .k-file-invalid-extension-wrapper,
-        .k-multiple-files-invalid-extension-wrapper,
-        .k-file-invalid-group-wrapper,
-        .k-multiple-files-invalid-group-wrapper {
-            color: $upload-icon-color;
-            border-color: $upload-icon-color;
-        }
-
-        // old rendering
-        .k-file-invalid-extension-wrapper,
-        .k-multiple-files-invalid-extension-wrapper,
-        .k-file-error .k-file-extension-wrapper,
-        .k-file-invalid-group-wrapper,
-        .k-multiple-files-invalid-group-wrapper,
-        .k-file-error .k-file-group-wrapper {
-            border-color: $upload-icon-color;
-        }
-
-        // old rendering
-        .k-file-extension-wrapper::before,
-        .k-multiple-files-extension-wrapper::before,
-        .k-file-invalid-extension-wrapper::before,
-        .k-multiple-files-invalid-extension-wrapper::before,
-        .k-file-error .k-file-extension-wrapper::before {
-            background-color: $upload-bg;
-            border-color: transparent transparent $upload-icon-color $upload-icon-color;
-        }
-
-        // old rendering
-        .k-multiple-files-extension-wrapper::after,
-        .k-multiple-files-invalid-extension-wrapper::after {
-            border-top-color: $upload-icon-color;
-            border-left-color: $upload-icon-color;
+        .k-file-icon-wrapper {
+            color: $kendo-upload-icon-color;
+            border-color: $kendo-upload-icon-color;
         }
 
         .k-file-size,
         .k-file-validation-message,
-        .k-file-information {
+        .k-file-summary {
             color: $subtle-text;
         }
 
-        .k-multiple-files-wrapper .k-file-information {
-            color: $upload-text;
+        .k-multiple-files-wrapper .kk-file-summary {
```

```suggestion
        .k-multiple-files-wrapper .k-file-summary {
```

double kk in selector.